### ISSUE TITLE
Shader dosen't render on realms

### ISSUE THREAD

**Issue by sheradrax** [McbeEringi/esbe-2g#7]
I've never ran into this issue with a shader. If I upload a map to the realm with the shader, it deactivates the shader. If i enable it on the realm instead, it doesn't render bust shows up in the active resource packs.

**Comment by tmzphoto** [comment]
Yes, I got exactly the same result. If I upload a world to realms with ANY shader, it doesn't render. If I copy the world back down, the shader is gone. I haven't been able to get any shader to work with realms or Xbox since the 1.13.x update, but shaders still work on PC. Most generic resource /texture packs will still work. I am starting to think that Microsoft has done something to "lock down" this functionality.

**Comment by sheradrax** [comment]
I was actually able to get the shader to work without any texture packs on
the realm.  It also worked once on a local game with a texture pack. It's
weird. I think it's a bug.
